launcher/FileSystem.cpp

```diff
@@ -316,8 +296,7 @@ QString DirNameFromString(QString string, QString inDir)
         if (num == 0) {
             dirName = baseName;
         } else {
-            dirName = baseName + QString::number(num);
-            ;
+            dirName = baseName + "(" + QString::number(num) + ")";
```

Comment:
at least on windows (e.g. when making a new text document) it also has a space between the brackets and text.
If this is not the case on other platforms, feel free to ignore this suggestion... or put it in a #if windows section
```suggestion
            dirName = baseName + " (" + QString::number(num) + ")";
```

Reply:
hmm, at least when i download things from Firefox into my Downloads folder, it doesn't add the space, so i don't know, i think there's not really a standard :/

Reply:
oh true, firefox doesn't add a space
well, then disregard my comment, leave it as it is. ^^

(for reference, this was my quick test on the desktop)
![image](https://user-images.githubusercontent.com/31988415/190020535-95513908-ffc5-498e-8cf3-a611db07f26b.png)